src/Rings/MPolyMap/MPolyQuo.jl

```diff
@@ -0,0 +1,95 @@
+################################################################################
+#
+#  Domains of type MPolyQuo
+#
+################################################################################
+
+function _check_imgs_quo(R, S::NCRing, imgs)
+  for i in 1:n, j in 1:(i - 1)
+    @req imgs[i] * imgs[j] == imgs[j] * imgs[i] "Images $i and $j do not commute"
+  end
+  gensI = gens(modulus(R))
+  for g in gensI
+    @req iszero(evaluate(g, imgs)) "Morphism is not well-defined"
```

Comment:
This `@req` macro seems to be a shortcut for writing
```suggestion
    iszero(evaluate(g, imgs)) || throw(ArgumentError("Morphism is not well-defined"))
```
correct?

Reply:
Yes.